### DISCUSSION TITLE
How to insert the regex to exclude URLs in Stats? // Wie muss der regex Eintrag aussehen um Seiten in den Stats auszuschließen?

### DISCUSSION THREAD

**Discussion by friese1971** [AndiLeni/statistics#125] [Q&A]
Ich habe Zugriffe auf die URL wo immer "alfa" vorkommt und das möchte ich Ausschließen....
xxxxxx.de/ALFA_DATA/alfacgiapi/index.php?bx=0e215962017
Wie muss der Eintrag aussehen?

**Comment by AndiLeni** [comment]
Dafür wird die preg_match Funktion genutzt (https://www.php.net/manual/de/function.preg-match.php).
Wenn alle URLs ignoriert werden sollen in denen "alfa" vorkommt, reicht es, `/alfa/` als regex zu nutzen.

**Reply by friese1971** [reply]
Ach... dann schreibe ich nur /alfa/ da rein? Mehr nicht?... das ist ja einfach :-)

**Reply by AndiLeni** [reply]
Ja genau. Pro Zeile kann man einen Regex Ausdruck angeben :-)

**Reply by friese1971** [reply]
Jetzt ist ja der Schrott schon in der DB. 
Kann ich aus der DB Tabelle rex_pagestats_visits_per_url die Einträge löschen?
Oder wie könnte ich das aufräumen um "echte" Zahlen zu bekommen?

**Reply by AndiLeni** [reply]
Du kannst die URLs aus pagestats_urlstatus und pagestats_visits_per_url löschen, aber vorher ein Backup machen!
Ich bin nicht sicher wie die Software da genau reagiert.
Aus den "Besuchen pro Tag" bekommst du sie leider nicht mehr raus, da diese Tabelle bereits aggregiert gespeichert wird und keine Informationen zu der besuchten URL hat.
